src/twisted/web/_flatten.py

```diff
@@ -381,14 +385,40 @@ async def _flattenTree(
 
     @return: A C{Deferred}-returning coroutine that resolves to C{None}.
     """
+    buf = []
+    bufSize = 0
+
+    # Accumulate some bytes up to the buffer size so that we don't annoy the
```

Comment:
very minor comment regarding the style.
Why have a comment and not a docstring ?

Reply:
Usually I don't put docstrings on inner functions since it is extremely unlikely that anyone will ever see them as docstrings and if they are docstrings then they are held in memory.  I don't feel too strongly either way though.